crates/rolldown/tests/fixtures/function/outro/iife/artifacts.snap

```diff
@@ -9,14 +9,14 @@ input_file: crates/rolldown/tests/fixtures/misc/outro/iife
 
 ```js
 (function() {
-
 "use strict";
 
 //#region main.js
 var main_default = "banner";
 
 //#endregion
-// outro
 return main_default;
+// outro
```

Comment:
This behavior is expected, [Rollup's REPL here](https://rollupjs.org/repl/?version=4.19.0&shareable=JTdCJTIyZXhhbXBsZSUyMiUzQW51bGwlMkMlMjJtb2R1bGVzJTIyJTNBJTVCJTdCJTIyY29kZSUyMiUzQSUyMmV4cG9ydCUyMGRlZmF1bHQlMjAnYmFubmVyJyUyMiUyQyUyMmlzRW50cnklMjIlM0F0cnVlJTJDJTIybmFtZSUyMiUzQSUyMm1haW4uanMlMjIlN0QlMkMlN0IlMjJjb2RlJTIyJTNBJTIyJTJGJTJGJTIwbWF0aHMuanMlNUNuJTVDbiUyRiUyRiUyMFRoaXMlMjBmdW5jdGlvbiUyMGlzbid0JTIwdXNlZCUyMGFueXdoZXJlJTJDJTIwc28lNUNuJTJGJTJGJTIwUm9sbHVwJTIwZXhjbHVkZXMlMjBpdCUyMGZyb20lMjB0aGUlMjBidW5kbGUuLi4lNUNuZXhwb3J0JTIwY29uc3QlMjBzcXVhcmUlMjAlM0QlMjB4JTIwJTNEJTNFJTIweCUyMColMjB4JTNCJTVDbiU1Q24lMkYlMkYlMjBUaGlzJTIwZnVuY3Rpb24lMjBnZXRzJTIwaW5jbHVkZWQlNUNuJTJGJTJGJTIwcmV3cml0ZSUyMHRoaXMlMjBhcyUyMCU2MHNxdWFyZSh4KSUyMColMjB4JTYwJTVDbiUyRiUyRiUyMGFuZCUyMHNlZSUyMHdoYXQlMjBoYXBwZW5zISU1Q25leHBvcnQlMjBjb25zdCUyMGN1YmUlMjAlM0QlMjB4JTIwJTNEJTNFJTIweCUyMColMjB4JTIwKiUyMHglM0IlNUNuJTVDbiUyRiUyRiUyMFRoaXMlMjAlNUMlMjJzaWRlJTIwZWZmZWN0JTVDJTIyJTIwY3JlYXRlcyUyMGElMjBnbG9iYWwlNUNuJTJGJTJGJTIwdmFyaWFibGUlMjBhbmQlMjB3aWxsJTIwbm90JTIwYmUlMjByZW1vdmVkLiU1Q253aW5kb3cuZWZmZWN0MSUyMCUzRCUyMCdjcmVhdGVkJyUzQiU1Q24lNUNuY29uc3QlMjBpbmNsdWRlRWZmZWN0JTIwJTNEJTIwZmFsc2UlM0IlNUNuaWYlMjAoaW5jbHVkZUVmZmVjdCklMjAlN0IlNUNuJTVDdCUyRiUyRiUyME9uJTIwdGhlJTIwb3RoZXIlMjBoYW5kJTJDJTIwdGhpcyUyMGlzJTIwbmV2ZXIlNUNuJTVDdCUyRiUyRiUyMGV4ZWN1dGVkJTIwYW5kJTIwdGh1cyUyMHJlbW92ZWQuJTVDbiU1Q3R3aW5kb3cuZWZmZWN0MSUyMCUzRCUyMCdub3QlMjBjcmVhdGVkJyUzQiU1Q24lN0QlMjIlMkMlMjJpc0VudHJ5JTIyJTNBZmFsc2UlMkMlMjJuYW1lJTIyJTNBJTIybWF0aHMuanMlMjIlN0QlNUQlMkMlMjJvcHRpb25zJTIyJTNBJTdCJTIyb3V0cHV0JTIyJTNBJTdCJTIyZm9ybWF0JTIyJTNBJTIyaWlmZSUyMiUyQyUyMm5hbWUlMjIlM0ElMjIlMjIlMkMlMjJvdXRybyUyMiUzQSUyMiUyRiUyRiUyMG91dHJvJTIyJTdEJTJDJTIydHJlZXNoYWtlJTIyJTNBdHJ1ZSU3RCU3RA==).